src/collections/blog/2022/2022-06-13-yash-gsoc-blogpost/index.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: "My Journey as a Contributor to GSoC Intern @Layer5"
+subtitle: "A Rollercoaster experience"
+date: 2022-06-10 08:00:00 -0630
+author: Yash Kamboj
+thumbnail: 
+category: Community
+description: "Aaditya Narayan Subedy's journey from a Contributor @Layer5 to getting an GSoC internship @Layer5"
```

Comment:
Templates are fun

Reply:
Missed that 😅😅

Reply:
@Abhishek-kumar09 Thank you for pointing that out  😅